manage.py

```diff
@@ -118,6 +97,85 @@ def analyze_all(self):
         )
         print('Batcher invocation successful!')
 
+    @staticmethod
+    def build():
+        """Build Lambda packages (saves *.zip files in terraform/)."""
+        lambda_build(TERRAFORM_DIR)
+
+    def deploy(self):
+        """Deploy BinaryAlert. Equivalent to test + build + apply + analyze_all."""
+        self.test()
+        self.build()
+        self.apply()
+        self.analyze_all()
+
+    def live_test(self):
+        """Upload an EICAR test file to BinaryAlert which should trigger a YARA match alert."""
+        bucket_name = '{}.binaryalert-binaries.{}'.format(
+            self._config['name_prefix'].replace('_', '.'), self._config['aws_region'])
+        test_filename = 'eicar_test_{}.txt'.format(uuid.uuid4())
+        s3_identifier = 'S3:{}:{}'.format(bucket_name, test_filename)
+
+        print('Uploading {}...'.format(s3_identifier))
+        bucket = boto3.resource('s3').Bucket(bucket_name)
+        bucket.put_object(
+            Body=EICAR_STRING.encode('UTF-8'),
+            Key=test_filename,
+            Metadata={'observed_path': test_filename}
+        )
+
+        print('File uploaded! Waiting for new Dynamo entry to appear...')
```

Comment:
nit: folks using this may not be familiar with AWS services/references, so can you change all string output references of `Dynamo` to `DynamoDB`to aid in readability/understanding?